angular/package.json

```diff
@@ -4,9 +4,14 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
+    "start:local": "ng serve --host hazmapper.local",
     "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
+    "test": "ng test --watch=false --browsers=ChromeHeadless",
+    "test:ci": "ng test --browsers=ChromeHeadless --progress=false --watch=false --code-coverage --source-map=false",
+    "lint": "npm run prettier && npm run lint:js && npm run lint:css",
+    "prettier": "prettier --write ./src",
+    "lint:js": "ng lint --fix",
```

Comment:
the `lint` and `prettier` command both have the fixing option ("--fix" and "--write").  so "npm run lint" ends up fixing any errors and returns 0 instead of returning an error.  maybe have another entry or two that just do the checking part.